components/DelegatingWidget/Input.tsx

```diff
@@ -4,7 +4,7 @@ import { useExplorerStore } from "hooks";
 import { useEffect, useMemo } from "react";
 import { useWindowSize } from "react-use";
 
-const Input = ({ transcoder, value, onChange, protocol, ...props }) => {
+const Input = ({ transcoder, value, onChange, protocol, treasury, ...props }) => {
```

Comment:
Since I'm new to the codebase, I'm not sure if this is the desired way to pass this information. There are multiple ways to do this. We can also use a hook or global state.

Reply:
This looks good to me. There are tons of areas in the codebase we can improve, especially around state mgmt. Any suggestions in future PRs are always welcome.